lib/trento_web/controllers/v1/personal_access_tokens_controller.ex

```diff
@@ -53,4 +53,33 @@ defmodule TrentoWeb.V1.PersonalAccessTokensController do
       })
     end
   end
+
+  operation :revoke_personal_access_token,
+    summary: "Revokes an existing Personal Access Token",
+    tags: ["Profile"],
+    parameters: [
+      jti: [
+        in: :path,
+        required: true,
+        type: %OpenApiSpex.Schema{
+          type: :string,
+          format: :uuid,
+          example: "550e8400-e29b-41d4-a716-446655440000"
+        }
+      ]
+    ],
+    responses: [
```

Comment:
Do we have a `not found` return?
I mean, it can happen that the coming jti doesn't exist or it is already removed.